Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -189,12 +189,12 @@ import BraintreeDataCollector
     /// If the user cancels out of the flow, the error code will be `.canceled`.
```

Comment:
```    
    /// Tokenize a PayPal request to be used with the Edit FI flow.
    ///
    /// On success, you will receive an instance of `BTPayPalAccountNonce`; on failure or user cancelation you will receive an error.
    /// If the user cancels out of the flow, the error code will be `.canceled`.```
    
We may want to update this comment, removing the references to tokenize and nonce.

Reply:
Good catch, think it caught them all here: https://github.com/braintree/braintree_ios/pull/1369/commits/a51c73d8925b0e874bdc2d1f191ec4f1ba742169